### ISSUE TITLE
Share Extension support

### ISSUE THREAD

**Issue by stevenleija** [ionic-team/capacitor#2416]
Support for the Share Extension would be great.  I found a cordova plugin https://github.com/j3k0/cordova-plugin-openwith but it's no longer being maintained in cordova.

**Comment by jcesarmobile** [comment]
Latest release of that plugin was done 1 month ago, so not sure why do you say it's not being maintained.

That plugin has hooks to add the target/share extension to the project, those kind on things are supposed to be done manually in Capacitor because Capacitor doesn't edit your native project.

**Comment by tyvdh** [comment]
Just tried to add this plugin to a Capacitor project and it doesn't work. Some instructions or a basic tutorial for getting this to work in a Capacitor project would be amazing! The build variables make it quite difficult to know how to replicate the install. Especially due to the several hooks.

**Comment by stevenleija** [comment]
> Just tried to add this plugin to a Capacitor project and it doesn't work. Some instructions or a basic tutorial for getting this to work in a Capacitor project would be amazing! The build variables make it quite difficult to know how to replicate the install. Especially due to the several hooks.

Correct.  I've tried it several different ways including start new test projects to see if it's conflicting with other plugins.  Still no luck.

**Comment by oudj001** [comment]
Anyone any updates on this? Open With plugin is unfortunately not adding the ShareExtension target to the project in Capacitor.

**Comment by jfbloom22** [comment]
I can verify that his plugin is not compatible.  It is using hooks to update the iOS project and add a "share extension" to the app's capabilities.  I found it is possible to generate this share extension in a cordova app and then copy it over to both the App and the Pods like this:  https://www.dropbox.com/s/pd93brz7w1sr3f6/Screen%20Shot%202020-10-16%20at%2010.27.04%20AM.png?dl=0.  I had to first add a "share extension" in Xcode and then replace the contents with the one I generated in the cordova app.  I was able to get the share option to show in iOS, but every time I ran "npx cap copy" I had to re-do my configuration.  If this plugin was more important to the solution I am building it would be fun to build a capacitor version of this share plugin.
